client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-meta-data/motion-meta-data.component.html

```diff
@@ -41,12 +41,23 @@ <h4>{{ 'Supporters' | translate }}</h4>
         <!-- show supporters (TODO: open in dialog) -->
         @if (motion.hasSupporters()) {
             <button mat-button type="button" (click)="openSupportersDialog()">
-                {{ motion.supporters.length }} {{ 'supporters' | translate }}
+                {{ motion.supporters.length }}{{ ' supporters' | translate }}
             </button>
+            @if (validSupporters > 0 && canManage && (allowSupporters$ | async)) {
+                <span class="warn">{{ 'of which ' + validSupporters + ' are not permissable' | translate }}</span>
```

Comment:
This will not translate.

Reply:
We used sometimes %num% and replace for such translate problem.